libbpf-rs/src/skeleton.rs

```diff
@@ -313,3 +316,26 @@ impl<'a> Drop for ObjectSkeletonConfig<'a> {
         unsafe { Box::from_raw(self.inner.obj) };
     }
 }
+
+pub trait SkelBuilder<'a> {
```

Comment:
Can we please add documentation for these traits and their methods?

Reply:
Ok, will do